app/util/files/FileGrid.js

```diff
@@ -0,0 +1,80 @@
+Ext.define('CpsiMapview.util.files.FileGrid',
```

Comment:
Add as a view rather than in util folder

Reply:
done